### PR TITLE
community: added `partners/package-name` folders

### PR DESCRIPTION
**Pull request by leo-gan**
Added references to new integration packages from Google, by adding subfolders to `partners/`.

### CODE REVIEW

File: libs/partners/google-alloydb-pg/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+placeholder
```
Comment (baskaryan) [inline libs/partners/google-alloydb-pg/README.md:1]: could we give these pkg name as title instead?
```markdown
# langchain-google-alloydb-pg
```

Comment (leo-gan) [reply libs/partners/google-alloydb-pg/README.md:1]: Sure. Wouldn't it be confusing? Probably the same as `placeholder` :)

Comment (leo-gan) [reply libs/partners/google-alloydb-pg/README.md]: @baskaryan It is ready.

**Comment by leo-gan** [comment]
@baskaryan Thanks!!

**Comment by efriis** [comment]
@leo-gan this breaks the api docs build, and the folders here should not be the source of truth of ecosystem packages. The only reason there are some here is to be "tombstones" for packages that used to be here and moved.

Reverting

**Pull request merged**